lib.php

```diff
@@ -335,7 +335,15 @@ function local_annoto_extend_settings_navigation(settings_navigation $settingsna
 
     // Check and create LTI external tool.
     require_once($CFG->dirroot . '/mod/lti/locallib.php');
-    $lti = lti_get_tool_by_url_match($settings->toolurl);
+    // Will return all available tools by domain from toolurl.
+    $possibletools = lti_get_tools_by_url($settings->toolurl, LTI_TOOL_STATE_CONFIGURED, null);
```

Comment:
The approach should be:
1. Get the Annoto LTI tools by using the correct domain.
2. Find LTI 1.3 tool that points to the dashboard (make sure it's not the assignments embed, but the dashboard tool)
3. If not found LTI 1.3, find LTI 1.1